content/en/database_monitoring/setup_mongodb/mongodbatlas.md

```diff
@@ -67,7 +67,7 @@ To monitor your MongoDB Atlas Cluster, you must install and configure the Datado
 
 Applications usually connect to MongoDB Atlas using an SRV connection string, but the Datadog Agent must connect directly to the individual MongoDB instance being monitored. If the Agent connects to different MongoDB instance while it is running (as in the case of failover, load balancing, and so on), the Agent calculates the difference in statistics between two hosts, producing inaccurate metrics.
 
-To get the individual MongoDB instance hostname and port, you can use the `dig` command to resolve the SRV connection string:
+To get the individual MongoDB instance hostname and port, you can use network utility command line tools like `dig` command in Linux or `nslookup` command in Windows to resolve the SRV connection string.
```

Comment:
nit: Consider removing the word "command" after the command.

Example: "...like `dig` in Linux or `nslookup` in Windows ..."